rbx_binary/src/tests/models.rs

```diff
@@ -42,4 +42,5 @@ binary_tests! {
     cframe_special_cases,
     cframe_case_mixture,
     three_unique_parts,
+    three_brickcolorvalues,
```

Comment:
Huh. I didn't know this was a type.

Reply:
I didn't either until I started implementing this! It ended up being a happy coincidence.